### PR TITLE
Adding functionality for get order by partner order id.

### PR DESCRIPTION
**Pull request by chelseawinfree**

### CODE REVIEW

File: symantecssl/response_models.py [hunk 1]
```diff
@@ -252,6 +257,50 @@ def deserialize(cls, xml_node):
...
 
+class Vulnerabilities(list):
+
+    def __init__(self, details_to_add=[]):
+        self.extend(details_to_add)
+
+    @classmethod
+    def deserialize(cls, xml_node):
+        """Deserializes the Vulnerabilities section in the response.
+
```
Comment (jmvrbanac) [inline symantecssl/response_models.py]: You don't need the extra newline here

File: symantecssl/request_models.py [hunk 2]
```diff
@@ -548,3 +548,50 @@ def set_order_parameters(
...
+        """Serializes the get order by partner order ID.
+
+        The request model for the GetOrderByPartnerOrderID call in the Symantec
+        SOAP XML API. Serializes all related sections to this request model.
+
+        This will serialize the following:
+            Query Request Header
+            Query Options
+
+
```
Comment (jmvrbanac) [inline symantecssl/request_models.py]: You don't need the extra newline here

File: symantecssl/request_models.py [hunk 3]
```diff
@@ -454,6 +410,50 @@ def set_query_options(
...
+        """Serializes the modified orders request.
+
+        The request model for the GetModifiedOrders call in the Symantec
+        SOAP XML API. Serializes all related sections to this request model.
+
+        This will serialize the following:
+            Query Request Header
+            Query Options
+
+
```
Comment (jmvrbanac) [inline symantecssl/request_models.py]: You don't need the extra newline here

**Comment by jmvrbanac** [comment]
Other than a couple minor stylistic docstring changes, it looks good!

**Comment by sheyman** [comment]
LGTM

**Comment by jmvrbanac** [comment]
:+1:

**Pull request merged**